x-pack/test/apm_api_integration/tests/entities/logs/service_logs_error_rate_timeseries.spec.ts

```diff
@@ -165,7 +165,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             });
 
             expect(response.status).to.be(200);
-            expect(first(response.body.currentPeriod?.['my-service'])?.y).to.be(0.5);
```

Comment:
hmmmm this doesn't look right. cc: @kpatticha 

Is this because we now use `asInteger?`

Reply:
No, it’s because we’re no longer calculating the value relative to the total logs.

Previously, it was `errorLogs / totalLogs`, but now it’s just `errorLogs`. So, the expected value in the test is 1.

Reply:
cool, thanks for the explanation.